tests/contrib/config.py

```diff
@@ -40,6 +40,14 @@
     'database': os.getenv('TEST_MYSQL_DATABASE', 'test'),
```

Comment:
I can't comment on the file directly (thanks github) but can we delete the `tests/.DS_Store` introduced above?

Reply:
removed it